spec/models/document_spec.rb

```diff
@@ -321,158 +344,80 @@ def make_empty_temp_file
     end
   end
 
-  describe '#document#path' do
-    let(:document) do
-      create :document, document_file_name: 'test_file.doc', converted_preview_document_file_name: 'test_file.doc.pdf'
-    end
-    let(:id_partition) { format('%09d', document.id).scan(/\d{3}/).join('/') }
-    let(:filename) { document.document_file_name }
-
-    before do
-      stub_const 'PAPERCLIP_STORAGE_PATH', 'public/assets/test/images/:id_partition/:filename'
-    end
+  describe '#document' do
+    describe '#path' do
+      subject { document.document.path }
 
-    context 'without an Active Storage attachment' do
-      it 'has a path based on the filename' do
-        expect(document.document.path).to eq "public/assets/test/images/#{id_partition}/#{filename}"
+      let(:document) do
+        create :document, document_file_name: 'test_file.doc', converted_preview_document_file_name: 'test_file.doc.pdf'
       end
-    end
-
-    context 'with an Active Storage attachment in disk storage' do
-      require 'active_storage/service/disk_service'
+      let(:id_partition) { format('%09d', document.id).scan(/\d{3}/).join('/') }
+      let(:filename) { document.document_file_name }
 
       before do
-        ActiveStorage::Attachment.connection.execute(<<~SQL)
-          INSERT INTO active_storage_blobs (key, filename, content_type, metadata, byte_size, checksum, created_at)
-            VALUES ('test_key_original', 'test_file.doc', 100, '{}', 100, 'abc==', NOW())
-        SQL
-        ActiveStorage::Attachment.connection.execute(<<~SQL)
-          INSERT INTO active_storage_attachments (name, record_type, record_id, blob_id, created_at)
-            VALUES ('document', 'Document', #{document.id}, LASTVAL(), NOW())
-        SQL
-        ActiveStorage::Attachment.connection.execute(<<~SQL)
-          INSERT INTO active_storage_blobs (key, filename, content_type, metadata, byte_size, checksum, created_at)
-            VALUES ('test_key_preview', 'test_file.doc.pdf', 100, '{}', 100, 'abc==', NOW())
-        SQL
-        ActiveStorage::Attachment.connection.execute(<<~SQL)
-          INSERT INTO active_storage_attachments (name, record_type, record_id, blob_id, created_at)
-            VALUES ('converted_preview_document', 'Document', #{document.id}, LASTVAL(), NOW())
-        SQL
-
-        service = ActiveStorage::Service::DiskService.new(root: '/root/')
-        allow(ActiveStorage::Blob).to receive(:service).and_return(service)
+        stub_const 'PAPERCLIP_STORAGE_PATH', 'public/assets/test/images/:id_partition/:filename'
       end
 
-      it 'has the path for disk storage' do
-        expect(document.document.path).to eq '/root/te/st/test_key_original'
+      context 'without an Active Storage attachment' do
+        it { expect(document.document.path).to eq "public/assets/test/images/#{id_partition}/#{filename}" }
```

Comment:
As you have set a `subject` did you want to do:

```suggestion
        it { is_expected.to eq "public/assets/test/images/#{id_partition}/#{filename}" }
```

Reply:
yes, i meant to but was in a rush (school run 😄 )